src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -681,6 +773,30 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
         --mdc-icon-size: 30px;
       }
 
+      .controls ha-icon-button.seek-button {
+        position: relative;
+      }
+      .controls ha-icon-button.seek-button::after {
+        content: "${SEEK_BUTTON_INCREMENT}";
+        pointer-events: none;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        font-size: 0.7em;
+        font-weight: bold;
+      }
```

Comment:
There are no fitting icons for this in MDI, so I went with using regular refresh/restore icons and adding a pseudo-element in the center that displays the increment.